src/docs/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import MintButton from 'components/mint-button';
+import DocsEntry from './docs-entry';
+import DocsBlock from './docs-block';
+
+export default (props) => {
```

Comment:
How exactly React is used here? Is it a component or just a function which would be put to render afterwards?